app/src/main/java/com/imdbmovieapp/data/local/entity/MovieEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.imdbmovieapp.data.local.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "movieTable")
+data class MovieEntity(
+    val title: String = "",
+    val year: String = "",
+    val genre: String = "",
+    val image: String = "",
+    val isFavorite: Int = 0,
+    @PrimaryKey
+    val id: Int? = null,
```

Comment:
usually we write id first

Reply:
i think you dont need this default values, in some value is optional make it nullable